src/Http/Response.php

```diff
@@ -1983,6 +1983,42 @@ public function withLength($bytes)
         return $this->withHeader('Content-Length', (string)$bytes);
     }
 
+    /**
+     * Create a new response with the Link header set.
+     *
+     * ### Examples
+     *
+     * ```
+     * $response = $response->withAddedLink('http://example.com?page=1', ['rel' => 'prev'])
+     *     ->withAddedLink('http://example.com?page=3', ['rel' => 'next']);
+     * ```
+     *
+     * Will generate:
+     *
+     * ```
+     * Link: <http://example.com?page=1>; rel="prev"
+     * Link: <http://example.com?page=3>; rel="next"
+     * ```
+     *
+     * @param string $url link-value
```

Comment:
Can we have a better descriptive statement instead of `link-value` please? :)  Same for options param.

Reply:
And `@since 3.6.0` tag.

Reply:
hmmm :anguished:
Sorry, I can't think of good ideas...
If revision is required, I want you to help me.